deployment/.env.example

```diff
@@ -3,3 +3,6 @@ MQTT_PASSWORD=sample_password
 DOCKER_INFLUXDB_INIT_USERNAME=sample_user
 DOCKER_INFLUXDB_INIT_PASSWORD=sample_password
 DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=sample_token
+GF_SECURITY_ADMIN_USER=sample_user
```

Comment:
Where are these 3 variables used?
Please also add them the the containers, so they will be shown as missing if .env is missing.

Reply:
https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#override-configuration-with-environment-variables - they are not used in other places of the code but we need them to set different user & pass

Reply:
INFLUX_REMOTE_URL is used for the datasource setup